code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -334,6 +334,11 @@
 		icon_state = "hair_poofy"
 		gender = FEMALE
 
+	poofy2
```

Comment:
I looked at this file once when doing a batch of absolute pathing fixes and concluded they all had to be relatively pathed.